cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackend.java

```diff
@@ -66,42 +67,45 @@ void createCollection(String dbName, String collectionName, long collectionsSize
      * @throws Exception
      */
     void insertContextDataRaw(String dbName, String collectionName, ArrayList<Document> aggregation) throws Exception;
-
+    
     /**
-     * Inserts a new document in the given aggregated collection within the given database (row-like mode).
+     * Inserts pre-aggregated numeric context data in the related aggregated collction for the resolutions given.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1426/commits/a3bb342105b0c86818d833b69735985f450f02f8